projects/1.16/assets/chemlib/chemlib/lang/zh_cn.json

```
@@ -1 +1,407 @@
+  "item.chemlib.element_silver": "银",
+  "item.chemlib.element_cadmium": "镉",
+  "item.chemlib.element_indium": "铟",
+  "item.chemlib.element_tin": "锡",
+  "item.chemlib.element_antimony": "锑",
+  "item.chemlib.element_tellurium": "碲",
+  "item.chemlib.element_iodine": "碘",
+  "item.chemlib.element_xenon": "氙",
+  "item.chemlib.element_cesium": "铯",
+  "item.chemlib.element_barium": "钡",
+  "item.chemlib.element_lanthanum": "镧",
+  "item.chemlib.element_cerium": "铈",
+  "item.chemlib.element_praseodymium": "镨",
+  "item.chemlib.element_neodymium": "钕",
+  "item.chemlib.element_promethium": "钷",
+  "item.chemlib.element_samarium": "钐",
+  "item.chemlib.element_europium": "铕",
+  "item.chemlib.element_gadolinium": "钆",
+  "item.chemlib.element_terbium": "铽",
+  "item.chemlib.element_dysprosium": "镝",
+  "item.chemlib.element_holmium": "钬",
+  "item.chemlib.element_erbium": "铒",
+  "item.chemlib.element_thulium": "铥",
+  "item.chemlib.element_ytterbium": "镱",
+  "item.chemlib.element_lutetium": "镥",
+  "item.chemlib.element_hafnium": "铪",
+  "item.chemlib.element_tantalum": "钽",
+  "item.chemlib.element_tungsten": "钨",
+  "item.chemlib.element_rhenium": "铼",
+  "item.chemlib.element_osmium": "锇",
+  "item.chemlib.element_iridium": "铱",
+  "item.chemlib.element_platinum": "铂",
+  "item.chemlib.element_gold": "金",
+  "item.chemlib.element_mercury": "汞",
+  "item.chemlib.element_thallium": "铊",
+  "item.chemlib.element_lead": "铅",
+  "item.chemlib.element_bismuth": "铋",
+  "item.chemlib.element_polonium": "钋",
+  "item.chemlib.element_astatine": "砹",
+  "item.chemlib.element_radon": "氡",
+  "item.chemlib.element_francium": "钫",
+  "item.chemlib.element_radium": "镭",
+  "item.chemlib.element_actinium": "锕",
+  "item.chemlib.element_thorium": "钍",
+  "item.chemlib.element_protactinium": "镤",
+  "item.chemlib.element_uranium": "铀",
+  "item.chemlib.element_neptunium": "镎",
+  "item.chemlib.element_plutonium": "钚",
+  "item.chemlib.element_americium": "镅",
+  "item.chemlib.element_curium": "锔",
+  "item.chemlib.element_berkelium": "锫",
+  "item.chemlib.element_californium": "锎",
+  "item.chemlib.element_einsteinium": "锿",
+  "item.chemlib.element_fermium": "镄",
+  "item.chemlib.element_mendelevium": "钔",
+  "item.chemlib.element_nobelium": "锘",
+  "item.chemlib.element_lawrencium": "铹",
+  "item.chemlib.element_rutherfordium": "𬬻",
+  "item.chemlib.element_dubnium": "𬭊",
+  "item.chemlib.element_seaborgium": "𬭳",
+  "item.chemlib.element_bohrium": "𬭛",
+  "item.chemlib.element_hassium": "𬭶",
+  "item.chemlib.element_meitnerium": "鿏",
+  "item.chemlib.element_darmstadtium": "𫟼",
+  "item.chemlib.element_roentgenium": "𬬭",
+  "item.chemlib.element_copernicium": "鿔",
+  "item.chemlib.element_nihonium": "鉨",
+  "item.chemlib.element_flerovium": "𫓧",
+  "item.chemlib.element_moscovium": "镆",
+  "item.chemlib.element_livermorium": "𫟷",
+  "item.chemlib.element_tennessine": "Tennessine",
+  "item.chemlib.element_oganesson": "Oganesson",
```

```suggestion
  "item.chemlib.element_tennessine": "[[石田]]",
  "item.chemlib.element_oganesson": "[[奥气]]",
```
https://github.com/CFPAOrg/Minecraft-Mod-Language-Package/pull/1996#discussion_r891195304